client/src/ui-components/ShoppingCartComponent/ShoppingCartOrderSummary.js

```diff
@@ -0,0 +1,153 @@
+import "bootstrap/dist/css/bootstrap.min.css";
+import "./ShoppingCartOrderSummaryCSS.css";
+import React from "react";
+import { Container, Col, Row, Card, Form, FloatingLabel} from "react-bootstrap";
+import  Divider from "../Divider/Divider";
+import Button from "../Button/Button";
+import { OrderRecapRow } from "./OrderRecapRow";
+import { useLocation, useHistory } from "react-router-dom";
+
+
+function checkboxShipmentComponent(props){
+    return <div className="form-check">
+                <input onChange={()=>{
+                    if(props.deliveryType==="Pickup" || props.deliveryType===""){//i.e. if from "pickup" to "shipment", then clear the address field
+                        props.setDeliveryAddress("")
+                        props.setDeliveryFee((prev)=>{
+                            props.setAmount(props.amount+20);
+                            return 20;
```

Comment:
Use a variable" like props.deliveryFee like you did  ` <Row className='d-flex justify-content-between px-2 py-1'>
                    <Col>Fees</Col>
                    <Col className='align-end'>{props.deliveryFee + " €"}</Col>
                </Row>` here